### PR TITLE
removed r

### PR DESCRIPTION
**Pull request by ayesha-firdaus**
## Description
I have removed 'r' from line 31 

**Fixes** #5799

### CODE REVIEW

File: docs/data-integrations/firebird.mdx [hunk 1]
```diff
@@ -13,10 +13,10 @@ This handler is implemented using the `fdb` library, the Python driver for Fireb
 
 The required arguments to establish a connection are as follows:
 
-* `host` is the host name or IP address of the Firebird server.
-* `database` is the port to use when connecting with the Firebird server.
-* `user` is the username to authenticate the user with the Firebird server.
-* `password` is the password to authenticate the user with the Firebird server.
+- `host` is the host name or IP address of the Firebird server.
```
Comment (martyna-mindsdb) [inline docs/data-integrations/firebird.mdx:16]: You changed `*` into `-`.

Please change it back to `*` for all four points.

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@ayesha-firdaus 
Please see the review comment above.

Also, please sign our CLA as mentioned above.

File: docs/data-integrations/firebird.mdx (hunk 1 above)
Comment (ayesha-firdaus) [reply docs/data-integrations/firebird.mdx:16]: For more clarification do I have to change the 16th point + - to  + *

**Pull request closed without merging**